guides/doc-Administering_Red_Hat_Satellite/topics/log-files-provided-by-satellite.adoc

```diff
@@ -10,15 +10,22 @@
 [options="header"]
 |===
 | Log File Directories | Description of Log File Content
+ifndef::foreman-el,foreman-deb[]
 | `/var/log/candlepin` | Subscription management
+| `/var/log/tomcat` | Candlepin webservice logs
+endif::[]
 | `/var/log/foreman` | Foreman
 | `/var/log/foreman-proxy` | Foreman proxy
 | `/var/log/httpd` | Apache HTTP server
 | `/var/log/foreman-installer` | Installer
 | `/var/log/puppet` | Configuration management
```

Comment:
This changed to `/var/log/puppetlabs/puppet` when we moved to Puppet 4 in Satellite 6.3.